app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/SiteHeader.scala.html

```diff
@@ -29,6 +29,7 @@
   Seq(
     NavigationItem(content = Text("Applications"), href = Some(gatekeeperConfig.applicationsPageUri)),
     NavigationItem(content = Text("Developers"), href = Some(gatekeeperConfig.developersPageUri)),
+    NavigationItem(content = Text("Terms of use"), href = Some(uk.gov.hmrc.apigatekeeperapprovalsfrontend.controllers.routes.TermsOfUseInvitationController.page.url)),
```

Comment:
Could this be `gatekeeperConfig.termsOfUsePageUri`

Reply:
Well, it could be, but as the URL is actually in this repo (whereas the others are in api-gatekeeper-frontend or the email one) then might as well use the actual URL as we know it.